src/states/common/relocated_not_established.rs

```diff
@@ -29,7 +29,9 @@ pub trait RelocatedNotEstablished: Relocated {
         pub_id: &PublicId,
     ) -> Result<(), RoutingError> {
         match self.peer_mgr().get_peer(pub_id).map(Peer::state) {
-            Some(PeerState::Connected) | Some(PeerState::Proxy) => return Ok(()),
+            Some(PeerState::Node { .. }) | Some(PeerState::Connected) | Some(PeerState::Proxy) => {
```

Comment:
Proxy is gone now.

Reply:
Should be - I can try to remove as much of the Peer Manager as possible, too.